### ISSUE TITLE
[Bug]: window.Livewire.restart() is not a function

### ISSUE THREAD

**Issue by james2doyle** [swup/livewire-plugin#2]
### What did you expect? 🧐

Undefined methods should not be called 

### What actually happened? 😵‍💫

![image](https://github.com/swup/livewire-plugin/assets/1425304/39696743-bf32-4c72-be68-e6e85f2e13ad)
This function being called does not exist

### Swup and plugin versions 📝

- swup: 4.6.1
- livewire-plugin: 2.0.0

### Relevant log output 🤓

```shell
Hooks.ts:440 Error in hook 'content:replace': TypeError: window.Livewire.restart is not a function
    at i.refreshLivewireComponents (index.js:47:19)
    at index.ts:38:27
    at new Promise (<anonymous>)
    at f (index.ts:37:13)
    at S.run (Hooks.ts:434:26)
    at S.call (Hooks.ts:358:14)
    at async j.T [as renderPage] (renderPage.ts:21:23)
    at async navigate.ts:196:6
run @ Hooks.ts:440
await in run (async)
call @ Hooks.ts:358
await in call (async)
T @ renderPage.ts:25
(anonymous) @ navigate.ts:196
```

### URL to minimal reproduction 🔗

This would require a hosted Laravel livewire app

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for related problems
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

**Comment by james2doyle** [comment]
Looks like the current solution to restarting all the components is: 

```js
Livewire.all().map((component) => {
    component.call('render'); // or 'restart'
});
```

https://github.com/livewire/livewire/issues/2334#issuecomment-759901196

If this sounds right, I can make a PR with the changes

**Comment by james2doyle** [comment]
```js
if (typeof window.Livewire.restart !== 'function') {
    console.info('Patching restart function...');
    window.Livewire.restart = () =>
        Livewire.all().map((component) => {
            // not always defined, depending on the component
            if (typeof component.call === 'function') {
                return component.call('restart');
            }
            // will be called when the component is mounted again
            if (typeof component.$wire?.$refresh === 'function') {
                component.$wire.$refresh();
            }
        });
}
```

I wrote this function to patch this now removed restart function. Seems to be working, and I see the endpoints being called when the navigation occurs

**Comment by daun** [comment]
Yes, a PR would be fantastic! It would probably also be nice to have a conditional that supports both versions to avoid releasing a major version of the plugin.

**Comment by james2doyle** [comment]
@daun Ok cool. My code above has a check for the original function and then just replaces it. I think in the real code it would just be an if/else instead and then call each one natively. 

How does that sound?

**Comment by daun** [comment]
Sounds perfect!

**Comment by hirasso** [comment]
Closed by #3
